ckanext/datagovuk/tests/test_package.py

```diff
@@ -131,77 +131,6 @@ def test_resource_create_standard_file_upload(self, app):
         assert 'datafile-date' in fields
         assert 'format' in fields
 
-
-    ## Tests for indexing the package
-
-    @mock.patch("pysolr.Solr", autospec=True)
-    def test_package_indexing_truncates_unsafe_fields(self, mock_solr, app):
-        with app.flask_app.test_request_context():
-            index = PackageSearchIndex()
-            try:
-                submitted = {
-                    "id": "test-index",
-                    "name": "monkey",
-                    "title": "Monkey",
-                    "state": "active",
-                    "private": False,
-                    "type": "dataset",
-                    "owner_org": None,
-                    "metadata_created": "2020-06-06T12:12:12.000000Z",
-                    "metadata_modified": "2020-06-06T12:12:12.000000Z",
-                    "notes": "something " + ("really " * 20000) + "long",
-                    "extras_foo": [
-                        "another " + ("really " * 20000) + "long field",
-                        {"something": "smaller"},
-                    ],
-                    "status": {
-                        "boo": 1,
-                        "bar": 0,
-                        "baz": [
-                            "short",
-                            "short",
-                            "l" + ("o" * 30000) + "ng",
-                        ],
-                    },
-                }
-                submitted["foo"] = deepcopy(submitted["extras_foo"])
-
-                expected = deepcopy(submitted)
-                expected["foo"][0] = expected["foo"][0][:15000]
-                expected["status"]["baz"][2] = expected["status"]["baz"][2][:15000]
-                # indexer alters these
-                expected["metadata_created"] = mock.ANY
-                expected["metadata_modified"] = mock.ANY
-
-                index.index_package(submitted)
-
-                assert len(mock_solr.return_value.add.mock_calls) == 1
-                assert len(mock_solr.return_value.add.mock_calls[0][2]["docs"]) == 1
-
-                actual = mock_solr.return_value.add.mock_calls[0][2]["docs"][0]
-                # indexer adds a bunch of keys we don't care about and can't predict
-                actual_common = {
-                    k: v
-                    for k, v in actual.items()
-                    if k in expected
-                }
-
-                # status is coming back as str type rather than json
-                # set it back to json for full comparison
-                if type(actual_common['status']) is str:
-                   actual_common['status'] = json.loads(actual_common['status'])
-
-                assert expected == actual_common
-
-                try:
-                    # data_dict must still be valid json
-                    json.loads(actual["data_dict"])
-                except Exception as e:
-                    self.fail(e)
-            finally:
-                index.clear()
-
-
```

Comment:
Unclear why we're taking out this test?

Reply:
It was probably taken out during the upgrade work as it was failing, I've now added it back in with a fix for the data